### PR TITLE
[Serializer] Update changelog about the new $encoderIgnoredNodeTypes arg in XmlEncoder contrustor

### PR DESCRIPTION
**Pull request by maidmaid**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | https://github.com/symfony/symfony/pull/27926
| License       | MIT
| Doc PR        | /

### CODE REVIEW

File: src/Symfony/Component/Serializer/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
 * `AbstractNormalizer::handleCircularReference` is now final, and receives two optional extra arguments: the format and the context
 * added support for XML comment encoding (encoding `['#comment' => ' foo ']` results `<!-- foo -->`)
+* added optional `int[] $encoderIgnoredNodeTypes` argument to `XmlEncoder::__construct`.
```
Comment (dunglas) [inline src/Symfony/Component/Serializer/CHANGELOG.md]: `... to configure node types to be ignored during encodage`?

Comment (maidmaid) [reply src/Symfony/Component/Serializer/CHANGELOG.md]: Yes, better. Done !

File: src/Symfony/Component/Serializer/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ CHANGELOG
 
 * `AbstractNormalizer::handleCircularReference` is now final, and receives two optional extra arguments: the format and the context
 * added support for XML comment encoding (encoding `['#comment' => ' foo ']` results `<!-- foo -->`)
+* added optional `int[] $encoderIgnoredNodeTypes` argument to `XmlEncoder::__construct` to configure node types to be
+  ignored during encodage.
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Serializer/CHANGELOG.md]: encoding :)

Comment (maidmaid) [reply src/Symfony/Component/Serializer/CHANGELOG.md]: Done (\cc @dunglas :smile:)

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @maidmaid.

**Pull request merged**